### ISSUE TITLE
Can we use small model like LLAMA to get layout?

### ISSUE THREAD

**Issue by hujunchao** [TonyLianLong/LLM-groundedDiffusion#3]
Sometimes, there is no way to use Chatgpt or GPT4. Can we use smaller model than GPT like llama to get layout?
If we can use small model to get layout, It will be more and more popular for this job.
Looking forward to your reply! Thanks!

**Comment by TonyLianLong** [comment]
Yes. I tried Vicuna (https://github.com/lm-sys/FastChat) and it also works. However, proprietary models (e.g. gpt-3.5-turbo) are still better.

**Comment by TonyLianLong** [comment]
I tried LLaMA-2-7b and it seems to work:

<img width="1490" alt="image" src="https://github.com/TonyLianLong/LLM-groundedDiffusion/assets/1451234/a984fe91-a2fd-4381-9c0d-1468e0160099">

**Comment by hujunchao** [comment]
Oh, It is amazing! I will try it.
